scm-plugins/scm-svn-plugin/src/main/java/sonia/scm/repository/spi/SCMSvnDiffGenerator.java

```diff
@@ -374,9 +376,74 @@ public void displayPropsChanged(SvnTarget target, String revision1, String revis
       }
     }
 
-    displayPropertyChangesOn(useGitFormat ? getRelativeToRootPath(target, originalTarget1) : displayPath, outputStream);
+    if (useGitFormat) {
+      displayGitPropDiffValues(outputStream, propChanges, originalProps);
+    } else {
+      displayPropertyChangesOn(displayPath, outputStream);
+      displayPropDiffValues(outputStream, propChanges, originalProps);
+    }
+  }
+
+  private void displayGitPropDiffValues(OutputStream outputStream, SVNProperties diff, SVNProperties baseProps) throws SVNException {
+    for (Iterator changedPropNames = diff.nameSet().iterator(); changedPropNames.hasNext(); ) {
+      String name = (String) changedPropNames.next();
+      SVNPropertyValue originalValue = baseProps != null ? baseProps.getSVNPropertyValue(name) : null;
+      SVNPropertyValue newValue = diff.getSVNPropertyValue(name);
+
+      try {
+        byte[] originalValueBytes = getPropertyAsBytes(originalValue, getEncoding());
+        byte[] newValueBytes = getPropertyAsBytes(newValue, getEncoding());
```

Comment:
This does not handle binary properties well. I'd sugest we change `getPropertyAsBytes` here to something like the following to replace binary blobs with a simple text:
```java
  private byte[] getPropertyAsBytes(SVNPropertyValue value, String encoding, boolean replaceBinary) {
    if (value == null) {
      return null;
    }
    if (value.isString()) {
      try {
        return value.getString().getBytes(encoding);
      } catch (UnsupportedEncodingException e) {
        return value.getString().getBytes();
      }
    } else {
      return String.format("Binary value (%s bytes)", value.getBytes().length).getBytes();
    }
  }
```

Maybe we can leave the enconding out, because this seems not to be related to the property.

Reply:
I've applied the change.